bundles/org.openhab.io.hueemulation/pom.xml

```diff
@@ -14,7 +14,8 @@
   <name>openHAB Add-ons :: Bundles :: IO :: Hue Emulation Service</name>
 
   <properties>
-    <bnd.importpackage>org.glassfish.jersey.*;resolution:="optional"</bnd.importpackage>
+    <bnd.importpackage>org.glassfish.grizzly.*;resolution:="optional"</bnd.importpackage>
+    <bnd.importpackage>javax.ws.rs;version="[2.1,3.0)"</bnd.importpackage>
```

Comment:
Is that allowed and are these properties appended? I would guess that the second definition overrides the first.

Reply:
Comma's should be used instead:

https://github.com/openhab/openhab2-addons/blob/9bb69719f8b68897779b5a42a28f659edbc3a941/bundles/org.openhab.voice.marytts/pom.xml#L17